### PR TITLE
feat: Add theme toggle

### PR DESCRIPTION
**Pull request by knhash**
Add a navigation icon to toggle theme.

- loads both dark.css and light.css, toggles visibility via JavaScript
- use local storage to save theme state across pages

### CODE REVIEW

**Comment by knhash** [comment]
Apparently I had forgotten to add the theme icon to the repo. Fixed that. Attaching screenshots.

light
dark

**Comment by nielsenramon** [comment]
@knhash Looks awesome. Would it be easy to wrap it inside a setting in the config file? Not sure if I want to give it everyone by default!

**Comment by knhash** [comment]
@nielsenramon 
Theme toggle is now an option, defaulting to false.

**Comment by nielsenramon** [comment]
@knhash Nice work thanks!

File: _assets/javascripts/themetoggle.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  themer();
+}
+
+// Theme set
+function themer() {
+  var tone = localStorage.getItem('theme');
+  var light = document.getElementById("light");
+
+  if(tone == "dark"){
+    dark.disabled = false;
```
Comment (shen-david) [inline _assets/javascripts/themetoggle.js]: i'm not sure what this `dark` variable refers to? In any case, there's a bit of a flash whenever loading a new page in dark theme.

Comment (xRahul) [reply _assets/javascripts/themetoggle.js]: It looks like we are switching theme using JS and so until that JS loads, we are seeing the default theme that was set when building the website. There needs to be logic that reads localstorage before even the rendering happens.

**Pull request merged**